src/block/VanillaBlocks.php

```diff
@@ -1624,4 +1643,13 @@ private static function registerCauldronBlocks() : void{
 		self::register("lava_cauldron", new LavaCauldron(new BID(Ids::LAVA_CAULDRON, TileCauldron::class), "Lava Cauldron", $cauldronBreakInfo));
 		self::register("potion_cauldron", new PotionCauldron(new BID(Ids::POTION_CAULDRON, TileCauldron::class), "Potion Cauldron", $cauldronBreakInfo));
 	}
+
+	private static function registerBambooMosaicBlocks() : void{
+		$woodType = WoodType::BAMBOO();
+		$planksBreakInfo = new Info(BreakInfo::axe(2.0, null, 15.0));
+
+		self::register("bamboo_mosaic", new Planks(new BID(Ids::BAMBOO_MOSAIC), "Bamboo Mosaic", $planksBreakInfo, $woodType));
+		self::register("bamboo_mosaic_slab", new WoodenSlab(new BID(Ids::BAMBOO_MOSAIC_SLAB), "Bamboo Slab", $planksBreakInfo, $woodType));
+		self::register("bamboo_mosaic_stairs", new WoodenStairs(new BID(Ids::BAMBOO_MOSAIC_STAIRS), "Bamboo Stairs", $planksBreakInfo, $woodType));
```

Comment:
The names of these blocks are incorrect. Also, as I said previously, I think considering these as WoodType::BAMBOO is misleading. From an API perspective they will appear the same as actual bamboo stairs etc. which is misleading to plugin devs.